hugo_site/content/sprints.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Sprints"
+date: 2019-1-16T18:48:39+01:00
+draft: false
+type: "single"
+description: "Description of sprints "
+menu:
+  main:
+    name: "Sprints"
+    parent: "talks"
+---
+
+# Sprints
+
+So you have listened to all the exciting topics on django for three days and now feel motivated do something about it. 
+What could be a better motivation than contributing to the very open source software you just listened to?
+
+On Saturday and Sunday, you have the opportunity to do exactly that. We will have have fun, enjoy complimentary food and drinks, and oh yes we will have sprints and workshops. 
+This is your opportunity to contribute to Open Source software. Even if you haven't contributed before, this is also an option to take your first steps. In the sprint, you can contibute to Django as you see fit, and if you need help, there will a pool of experienced conributors to guide you.
+
+Besides Sprints we will also have workshops on various topics.
+
+The Sprints are free for all conference participants. However, as the Sprints venue does not have unlimited capacity, and we would like to prepare a precise amount of food and drinks, please keep your participation status updated on the Ticketholder Website (after buying your ticket).
+
+# Venue
+The Sprints will be arranged on the excellent indoor venue, with a big outdoor space at the heart of copenhagen. <br>
+<strong>Kulturhuset Indre By<br>
+Charlotte Ammundsens Plads 3<br> 
+1359 København K
+</strong>
+
+Closest train station:<strong> Nørreport</strong>. If you want to catch a train to airport on sunday evening straight from the venue, a metro service runs from there to Copenhagen Airport.
```

Comment:
You can use Markdown instead of HTML :) -- I think that the `<strong>` tag may be allowed, but it saves time and characters :)